xdsl/dialects/transform.py

```diff
@@ -207,21 +560,21 @@ def __init__(
         self,
         target: SSAValue,
         dynamic_sizes: Sequence[SSAValue],
-        static_sizes: DenseArrayBase | list[int] | None = None,
+        static_sizes: DenseArrayBase | list[int],
         interchange: DenseArrayBase | list[int] | None = None,
         scalable_sizes: DenseArrayBase | list[int] | None = None,
     ):
         if isinstance(static_sizes, list):
             static_sizes = DenseArrayBase.create_dense_int_or_index(
-                IndexType(), static_sizes
+                IntegerType(64), static_sizes
```

Comment:
Why is this change necessary?

Reply:
i64 is the correct format, I found. Index are not allowed. I didn't catch it at first as I hadn't initiated an instance using int's at first

Reply:
oh okay. Weird haha. Thanks for checking!